Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG KUBERNETES_VERSION=dev
 
 # Build environment
-FROM rancher/hardened-build-base:v1.17.13b7 AS build
+FROM rancher/hardened-build-base:v1.19.5b1 AS build
```

Comment:
I think there are a couple other references to bump

* https://github.com/rancher/rke2/blob/release-1.23/Dockerfile.windows#L8
* https://github.com/rancher/rke2/blob/release-1.23/.drone.yml#L14 (multiple occurrences in this file)